common/app/assets/assets.scala

```diff
@@ -39,6 +39,34 @@ class Assets(base: String, mapResource: String, useHashedBundles: Boolean = Conf
 
 }
 
+// turns a readable CSS class into a list of rules in short form from the atomic css file
+class CssMap(mapResource: String, useCssMap: Boolean = Configuration.assets.useHashedBundles) extends Logging {
+
+  lazy val lookup: Map[String, List[String]] = Get(cssMap(mapResource))
+
+  def apply(path: String): String = {
+    if (useCssMap) {
```

Comment:
i've updated the way it works to always output the atomic classes and the original one, so that special param is gone now anyway